src/CoreShop/Bundle/CoreBundle/Resources/translations/admin.de.yml

```diff
@@ -214,3 +214,5 @@ coreshop_solidify_variant_unit_definition_data_error_no_unit_definitions: 'Das P
 coreshop_solidify_variant_unit_definition_data_strategy_only_unit_definitions: 'Sind Sie sicher, dass Sie die Einheiten-Definitionen auf jede Variante überragen wollen? Dieser Vorgang kann nicht rückgängig gemacht werden! Nachdem Sie diesen Befehl ausgeführt haben, muss jede Variante separat verwaltet werden. Zudem werden alle Varianten, welche dem Produkt angehören, geschlossen. Nicht gespeicherte Änderungen gehen dabei verloren. Varianten, welche bereits Einheiten-Definitionen besitzen, werden übersprungen.'
 coreshop_solidify_variant_unit_definition_data_strategy_only_unit_definitions_and_qpr: 'Sind Sie sicher, dass Sie die Einheiten-Definitionen UND Stückmengen-Preisregeln auf jede Variante überragen wollen? Dieser Vorgang kann nicht rückgängig gemacht werden! Nachdem Sie diesen Befehl ausgeführt haben, muss jede Variante separat verwaltet werden. Zudem werden alle Varianten, welche dem Produkt angehören, geschlossen. Nicht gespeicherte Änderungen gehen dabei verloren. Varianten, welche bereits Einheiten-Definitionen oder Stückmengen-Preisregeln besitzen, werden übersprungen.'
 coreshop_solidify_variant_unit_definition_data_succesfully_applied: 'Alle Daten wurden erfolgreich auf die Varianten übertragen. Verarbeitete Datensätze: {0}.'
+coreshop_restore_inheritance: 'Vererbung wiederherstellen'
+coreshop_restore_inheritance_message: 'Sind Sie sicher das Sie die Store Eigenschaften des Produktes löschen und vom Mutter Element erben lassen wollen?'
```

Comment:
```suggestion
coreshop_restore_inheritance_message: 'Sind Sie sicher das Sie die Store-Eigenschaften des Produktes löschen und vom Eltern-Element erben lassen wollen?'
```

Reply:
👍 , but I was about to merge this PR, you can change the translation later on crowdin.com